packages/features/auth/signup/handlers/selfHostedHandler.ts

```diff
@@ -78,7 +80,26 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
         organizationSettings: true,
       },
     });
+
     if (team) {
+      const isInviteForOrganization = isOrganization({
+        team: {
+          metadata: team.metadata,
+        },
+      });
+      const isInviteForATeamInOrganization = team.parent;
+      const isCheckingUsernameInGlobalNamespace = !isInviteForOrganization && !isInviteForATeamInOrganization;
+
+      const teamMetadata = teamMetadataSchema.parse(team?.metadata);
```

Comment:
Getting a linting warning this isn't being used so let's remove it.

```suggestion
```

Reply:
Done